libbeat/filter/config.go

```diff
@@ -0,0 +1,14 @@
+package filter
+
+type DropFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type IncludeFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type FilterConfig struct {
+	DropFields    *DropFieldsConfig    `config:"drop_fields"`
```

Comment:
I would prefer `select_fields` over `drop_all_fields_except` as it is shorter and I think it also describes the issue.

In any case I would do the renaming directly in the same PR.